### ISSUE TITLE
error : type arguments cannot be applied to non-parameterized class 'NSArray'

### ISSUE THREAD

**Issue by sryze** [Microsoft/WinObjC#455]
When I compile code that has `NSArray<SomeClass *>`, for example:

```
@property (nonatomic) NSArray<NSNumber *> *someNumbers;
```

I get the following error:

```
error : type arguments cannot be applied to non-parameterized class 'NSArray'
```

Are lightweight generics currently supported?

**Comment by rajsesh-msft** [comment]
Should be available in the next release.

**Comment by rajsesh-msft** [comment]
fixed with 390d14661261562eb34ecf11543f0b7be0757318, available in 4/29 preview.
